test.js

```diff
@@ -223,6 +223,17 @@ test('should ignore digits', (t) => {
     }, t.ifErr)
 })
 
+test('should ignore times', (t) => {
+  t.plan(1)
+
+  retext()
+    .use(retextSpell, enGb)
+    .process('2:41pm')
+    .then((file) => {
+      check(t, file, [])
+    }, t.ifErr)
```

Comment:
```suggestion
    }, t.ifErr)

  retext()
    .use(retextSpell, enGb)
    .process("On my way! ETA 11:50!")
    .then((file) => {
      check(t, file, []);
    }, t.ifErr);
```

Another case for just a time